test/test_generate_pedigree.py

```diff
@@ -0,0 +1,128 @@
+"""
+methods for testing the sample-metadata API queries
+"""
+
+from copy import deepcopy
+
+from unittest.mock import patch
+
+import json
+import os
+
+from reanalysis.generate_pedigree import (
+    ext_to_int_sample_map,
+    get_clean_pedigree,
+    get_pedigree_for_project,
+)
+
+
+PWD = os.path.dirname(__file__)
+PROJECT = 'fake-project'
+INPUT = os.path.join(PWD, 'input')
+JSON_PED = os.path.join(INPUT, 'mock_pedigree.json')
+LOOKUP_PED = os.path.join(INPUT, 'mock_sm_lookup.json')
+
+SAMPLE_TO_CPG = {
+    'sam1': 'cpg1',
+    'sam2': 'cpg2',
+    'sam3': 'cpg3',
+}
+DIRTY_PED = [
+    {
+        'family_id': 'FAM1',
+        'individual_id': 'sam1',
+        'paternal_id': 'sam2',
+        'maternal_id': 'sam3',
+        'sex': 1,
+        'affected': 1,
+    },
+    {
+        'family_id': 'FAMX',
+        'individual_id': 'samx',
+        'paternal_id': 'samy',
+        'maternal_id': 'samz',
+        'sex': 1,
+        'affected': 1,
+    },
+]
+
+
+@patch('reanalysis.generate_pedigree.FamilyApi.get_pedigree')
```

Comment:
Could you put this in a unittest.TestCase block and test it in a github action?

Reply:
Resolving this as it's addressed in: https://github.com/populationgenomics/vigilant-winner/pull/3